.github/workflows/tests.yml

```diff
@@ -15,7 +15,9 @@ jobs:
       - name: Select Xcode 11.2
         run: sudo xcode-select -s /Applications/Xcode_11.2.app
       - name: Build and Test
-        run: bazelisk test --local_test_jobs=1 //...
+        run: |
+          bazelisk test --local_test_jobs=1 -- //... -//tests/mixed-source/ios/...
+          bazelisk test --local_test_jobs=1 --apple_platform_type=ios //tests/mixed-source/ios/...
```

Comment:
Still I do not know how to work around the fact that `--apple_platform_type=ios` is needed for some of the tests, and `--apple_platform_type=macos` is needed for others :/

Reply:
Wouldn't it work to put the macOS tests in `//tests/macos/...` and the iOS tests in `//tests/ios/...` so that you can pass the right `--apple_platform_type` arg?

Reply:
And placing the argument inside `tests/ios/.bacelrc`. Nice, did not think about that, I will try

Reply:
Nope, nested `bacelrc` files do not work. We will need to call `bazel test` twice with different arguments. I will reorganize the test folder as you mentioned